accepted/0000-expand-list-of-packlist-ignored-files.md

```diff
@@ -0,0 +1,37 @@
+# Expand list of ignored files
+
+## Summary
+
+Let's expand the default list of ignored files in [packlist](https://github.com/npm/npm-packlist).
+
+## Motivation
+
+A large population of npm users are concerned about package sizes and with the advent of the file explorer now available on [npmjs.com](https://www.npmjs.com/) we can now see a number of common files that are very intrinsic to the JS community that we could start ignoring from package bundles without too much friction to the larger ecosystem.
+
+## Detailed Explanation
+
+Expand the current list of ignored files to also ignore by default:
+
+- `.editorconfig` common plugins
+- `.gitattributes` and/or more git things
+- `.idea/` (or other editors similar configs/store/etc)
+- `.travis.yml`, `.github/` (and/or more ci services)
```

Comment:
FWIW, removing `.travis.yml`/`.github/` files would be a breaking change for some packages that use the CI configuration in order to provide users with useful messaging about supported platforms (where the supported platforms are defined as platforms that are under test). 

An example would be [this code in ember-cli](https://github.com/ember-cli/ember-cli/blob/v3.18.0/lib/utilities/platform-checker.js), which is used to emit warnings like:

> Node ${process.version} is no longer supported by Ember CLI. We recommend that you use the most-recent "Active LTS" version of Node.js. See https://git.io/v7S5n for details.

Or:

> Node ${process.version} is not tested against Ember CLI on your platform. We recommend that you use the most-recent "Active LTS" version of Node.js. See https://git.io/v7S5n for details.

Reply:
wow, TIL people do this.

altho in this case, https://github.com/ember-cli/ember-cli/blob/v3.18.0/.npmignore#L18 would ensure ember-cli is not broken by this change.

Reply:
> wow, TIL people do this.

Ya, our goal was to ensure we could use a single source of truth for this guidance. We could remove it I suppose, but it has come in helpful for things like new Node releases that are untested with their release.

> altho in this case, ember-cli/ember-cli@v3.18.0/.npmignore#L18 would ensure ember-cli is not broken by this change.

Ya, good point. As long as the `.npmignore` "wins" (including negation) this won't be an issue. Thank you for looking at that!